RELEASE_NOTES.rst

```diff
@@ -21,6 +21,51 @@
 
 .. towncrier release notes start
 
+Airflow 2.8.3 (2024-03-11)
+--------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+The smtp provider is now pre-installed when you install Airflow. (#37713)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Bug Fixes
+"""""""""
+- Add "MENU" permission in auth manager (#37881)
+- Fix external_executor_id being overwritten (#37784)
+- Make more MappedOperator members modifiable (#37828)
+- Set parsing context dag_id in dag test command (#37606)
+
+Miscellaneous
+"""""""""""""
+- Remove useless methods from security manager (#37889)
+- Upgrade google-ads version (#37787)
```

Comment:
Why is this in a core release?

Reply:
It has an entry in pyproject.toml and if it's not released in 2.8+, new providers might not be installable. What do you think?

Reply:
cc @potiuk , is this true?

Reply:
Assuming you aren't doing source installs, provider deps from the provider itself. We don't need these changes to support the "normal" provider path.

Reply:
Yes. It's right, it might not change "source" constraints, but what we really nead for v2*branch are `pypi` constraints. Sometimes it's easier to cherry-pick some provider changes and sometims it's just neecessary where such changes might cause conflict and prevent building the images and generating constraints - but they are not **strictly** necessary. It's a bit case-by-case - but if you just regular bump lower limit of a provider and it's not swapping `<0.4.5` with `>=0.4.5` for example, this is generally not needed to cherry-pick those.

Reply:
And case like pandas is different - because there the changes were in both Airflow and providers,. so that one should stay.